dockerfiles/start-cloudburst.sh

```diff
@@ -36,17 +36,16 @@ gen_yml_list() {
 cd $HYDRO_HOME/anna
 git remote remove origin
 git remote add origin https://github.com/$ANNA_REPO_ORG/anna
-while ! (git fetch -p origin)
-do
-  echo "git fetch failed, retrying"
-done
+git fetch -p origin
 git checkout -b brnch origin/$ANNA_REPO_BRANCH
 git submodule sync
 git submodule update
 
-cd client/python && python3.6 setup.py install --prefix=$HOME/.local
+cd client/python
+python3.6 setup.py install
```

Comment:
why are we removing this prefix?

Reply:
It was inconsistent. There was no need to be using `$HOME/.local` because it installed things in a different place than the default, which worked fine.